pkgs/development/node-packages/node-packages.json

```diff
@@ -5,6 +5,8 @@
 , "@bitwarden/cli"
 , "@hyperspace/cli"
 , "@nestjs/cli"
+, "@prettier/plugin-pug"
+, "@prettier/plugin-xml"
```

Comment:
`nodePackages` only supports packaging of command line tools and not libraries. This means that Prettier extensions currently can't be packaged in Nixpkgs as a standalone package. Even if you install `prettier` and `@prettier/plugin-pug` from `nodePackages` into the same environment, `prettier` won't be able to find `@prettier/plugin-pug` because they do not share a common `node_modules` directory.

If you want to use Prettier extensions, I believe you should install them using npm/node2nix/yarn2nix on a per-project basis.